nginx/assets/dashboards/plus_overview.json

```diff
@@ -260,56 +260,6 @@
         "background_color": "vivid_blue",
         "layout_type": "ordered",
         "widgets": [
-          {
```

Comment:
Removing `Requests per second` widget entirely since `nginx.requests.total` is already on dashboard, so there's no need to have extra widget.